nyc/people.py

```diff
@@ -64,49 +64,53 @@ def scrape(self):
                 else:
                     role = 'Council Member'
 
-                district = web.get('District', '').replace(' 0', ' ')
+                if web:
+                    district = web.get('District', '').replace(' 0', ' ')
+                else:
+                    district = ''
 
                 p.add_term(role,
                            'legislature',
                            district=district,
                            start_date=self.toDate(term['OfficeRecordStartDate']),
                            end_date=self.toDate(term['OfficeRecordEndDate']))
 
-                party = web.get('Political Party')
+                if web:  # None if member page was deleted
+                    party = web.get('Political Party')
```

Comment:
Do you need to explicitly set party to something, like an empty string, if web is None?

Reply:
No, this mirrors the original logic, where [we only add a party if we can get one from the web info](https://github.com/opencivicdata/scrapers-us-municipal/blob/a5096da67dc94719c7ca2812c6fff8c7182c56a3/nyc/people.py#L75-L81).